### PR TITLE
[risk=no][RW-4838] Rename submitDataUseAgreement to submitDUCC

### PR DESCRIPTION
**Pull request by jmthibault79**
One piece of RW-4838.  After a release I'll remove the now-deprecated version.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/ProfileController.java [hunk 1]
```diff
@@ -325,11 +325,17 @@ private DbUser initializeUserIfNeeded() {
...
       Integer dataUseAgreementSignedVersion, String initials) {
     DbUser user =
-        userService.submitDataUseAgreement(
-            userProvider.get(), dataUseAgreementSignedVersion, initials);
+        userService.submitDUCC(userProvider.get(), dataUseAgreementSignedVersion, initials);
+    return getProfileResponse(saveUserWithConflictHandling(user));
+  }
+
+  @Override
+  public ResponseEntity<Profile> submitDUCC(Integer duccSignedVersion, String initials) {
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/api/ProfileController.java:337]: alt: submitDataUserCodeOfConduct() ?

File: ui/src/testing/stubs/profile-api-stub.ts [hunk 2]
```diff
@@ -138,8 +138,8 @@ export class ProfileApiStub extends ProfileApi {
     });
   }
 
-  public submitDataUseAgreement(dataUseAgreementVersion: number) {
-    this.profile.dataUseAgreementSignedVersion = dataUseAgreementVersion;
+  public submitDUCC(duccVersion: number) {
+    this.profile.dataUseAgreementSignedVersion = duccVersion;
```
Comment (jmthibault79) [inline ui/src/testing/stubs/profile-api-stub.ts:142]: `dataUseAgreementSignedVersion` is a bigger change for later

File: api/src/main/java/org/pmiops/workbench/api/ProfileController.java (hunk 1 above)
Comment (freemabd) [reply api/src/main/java/org/pmiops/workbench/api/ProfileController.java:337]: I think either is fine.

**Review by freemabd — COMMENTED** [review]
Looks good.

**Review by freemabd — APPROVED** [review]

**Pull request merged**